types/module/module.go

```diff
@@ -278,11 +278,11 @@ func (m *Manager) RegisterInvariants(ir sdk.InvariantRegistry) {
 // RegisterRoutes registers all module routes and module querier routes
 func (m *Manager) RegisterRoutes(router sdk.Router, queryRouter sdk.QueryRouter, legacyQuerierCdc *codec.LegacyAmino) {
 	for _, module := range m.Modules {
-		if !module.Route().Empty() {
-			router.AddRoute(module.Route())
+		if r := module.Route(); !r.Empty() {
+			router.AddRoute(r)
 		}
-		if module.QuerierRoute() != "" {
-			queryRouter.AddRoute(module.QuerierRoute(), module.LegacyQuerierHandler(legacyQuerierCdc))
+		if r := module.QuerierRoute(); r != "" {
```

Comment:
Ditto - the two solutions are practically equivalent. Why growing the diff size then?

Reply:
Because the call was repeated. See the PR description.

Reply:
> Because the call was repeated.

...therefore its result was cached. See my previous comment.